doc/api-extensions.md

```diff
@@ -1274,6 +1279,10 @@ A number of new syscalls related container configuration keys were updated.
 * `security.syscalls.deny`
 * `security.syscalls.allow`
 
+## `container_syscall_filtering_no_offensive_syntax`
```

Comment:
api extensions should be added at the end of the file - also not sure whether we need one for this or not.